### ISSUE TITLE
Music keeps playing when changing to another genere until pressing stop button

### ISSUE THREAD

**Issue by Mathamphetematiker** [igorbrigadir/ishkurs-guide-dataset#9]
Well you can handle this behaviour by pressing the stop button like I wrote in the title, but it would be more native to simply stop it either when changing to another genre or when starting a track of another genre. I'm sure it wasnt like that (playing multiple tracks at a time) in original Ishkurs guide 2.5.

Anyway I am so glad someone still cares about the old v2.5, I was so sad when it disappeared from di.fm and when I have done some researches on it last time I didnt find your project here. And its almos perfect, except my two created issues, I even could translate it to german by downloading the files to run it local.

Thank you so much!

**Comment by igorbrigadir** [comment]
Thanks! Unfortunately same with #8 - this is more of an issue with `ruffle` so you could maybe try running that outside of the browser? or a different version of ruffle may work https://ruffle.rs/#downloads - either way, it's not something i can fix myself but i'll keep this here in case a new specific version of ruffle works, so i can make that the default on the github pages

**Comment by Mathamphetematiker** [comment]
Do you think its a good idea to post this two issues to the bugtracker of ruffle? They actually ask for people testing their favourite flash applications. Should I simply copy and paste the adress to your ishkurs guide v2.5 and explain how it used to be working? Do you think they need any additional information? Thx or quick reply btw.

**Comment by igorbrigadir** [comment]
Yeah might be worth doing that - https://github.com/ruffle-rs/ruffle/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml the ruffle version i'm using is the latest nightly from `<script src="https://unpkg.com/@ruffle-rs/ruffle"></script>` currently `2023-07-07` so i guess it's the "Self Hosted Version" Scroll bar not working is probably related to an existing open issue that's worth referencing too: https://github.com/ruffle-rs/ruffle/issues/4847
